Taskfile.yml

```diff
@@ -93,41 +93,58 @@ tasks:
           exit 1
         fi
 
+  start-faktory:
```

Comment:
@BigBlue79 - you can now use `task start-faktory` in this repo and it'll startup faktory-ent along with opslevel-runner pointed to it.

Reply:
At least one person has relayed to me that Faktory doesn't start up for them in local dev unless they also have redis installed, too. Which makes sense, my mental model of "embedded" was a little wrong, it's more "unshared". How hard would it be to add a redis install here? Or is it easier for me to just update docs?

Reply:
Add an install-redis task